data/modules/SearchRescue/SearchRescue.lua

```diff
@@ -292,6 +293,15 @@ end
 -- basic mission functions
 -- =======================
 
+local verifyCommodity = function (item)
```

Comment:
I don't like it, as it fixes the symptom instead of the underlying problem (which is that it gets lost in the first place)